multiraft/events.go

```diff
@@ -42,6 +49,7 @@ type EventCommandCommitted struct {
 type EventMembershipChangeCommitted struct {
 	GroupID    uint64
 	CommandID  string
+	Index      uint64
```

Comment:
Does the same comments apply here for CommandID & Index? Might at least add a comment indicating that the comments in EventCommandCommitted apply here to CommandID and Index.